### PR TITLE
chown -R `whoami`:admin /usr/local if necessary

### PR DESCRIPTION
**Pull request by croaky**
We link to this in the docs:

https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/El_Capitan_and_Homebrew.md

Can we safely automate this to provide a better experience?

### CODE REVIEW

**Comment by croaky** [comment]
@thoughtbot/shell

File: mac [hunk 1]
```diff
@@ -46,6 +46,16 @@ fi
 # shellcheck disable=SC2016
 append_to_zshrc 'export PATH="$HOME/.bin:$PATH"'
 
+if [ -d "/usr/local" ]; then
+  if ! sudo -u "$(whoami)" [ -r /usr/local ]; then
```
Comment (gabebw) [inline mac]: This line looks like it's using `sudo` to run `[ -r /usr/local ]` as...the current user.

How is this different from the command without sudo, i.e. `if ! [ -r /usr/local ]`

File: mac [hunk 2]
```diff
@@ -46,6 +46,16 @@ fi
...
+    sudo chown -R "$(whoami)":admin /usr/local
+  fi
+else
+  mkdir /usr/local
```
Comment (mike-burns) [inline mac]: Does this need root privs?

File: mac (hunk 1 above)
Comment (mike-burns) [inline mac]: We don't need this conditional; this command is always safe to run.

Comment (gabebw) [inline mac]: Good point.

Comment (croaky) [inline mac]: @mike-burns We ( @gabebw and I ) decided to keep the `-r /usr/local` check to improve the script's speed.

File: mac (hunk 2 above)
Comment (croaky) [inline mac]: Yes, thanks!

**Pull request merged**